keycloak/openid_client.go

```diff
@@ -48,7 +48,8 @@ type OpenidClient struct {
 }
 
 type OpenidClientAttributes struct {
-	PkceCodeChallengeMethod string `json:"pkce.code.challenge.method"`
+	PkceCodeChallengeMethod             string `json:"pkce.code.challenge.method"`
+	ExcludeSessionStateFromAuthResponse bool   `json:"exclude.session.state.from.auth.response,string"`
```

Comment:
We typically use `KeycloakBoolQuoted` for these annoying boolean attributes that are actually strings.  https://github.com/mrparkers/terraform-provider-keycloak/blob/master/keycloak/util.go#L10

Reply:
Thanks, i fix it :)